File: GARI_Umbria_2026-08-03/GiasNG/src/app/Model/attivita/risorse/RisorsaAcqua.ts
import { Risorsa } from './Risorsa';

export class RisorsaAcqua extends Risorsa {
    acqua: number;
    doseAcqua: DoseAcqua;


    constructor() {
        super();
        this.classType = 'RisorsaAcqua';
    }
}

export enum DoseAcqua {
    TOTALE,
    HA
}

File: GARI_Umbria_2026-08-03/DotNet/AgronicaCoreStd/AgronicaCoreModelsSTD/attivita/risorse/RisorsaAcqua.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace AgronicaCoreModelsSTD.attivita.risorse
{
    public class RisorsaAcqua : Risorsa {
        public decimal acqua { get; set; }
        public TipoDoseAcqua doseAcqua { get; set; }

        public enum TipoDoseAcqua
        {
            TOTALE,
            HA
        }
        public RisorsaAcqua()
        {
            classType = costanti.ClassType.RisorsaAcqua;
        }
    }
}
